engine/src/main/scala/cromwell/webservice/routes/wes/WesRouteSupport.scala

```diff
@@ -108,6 +114,62 @@ trait WesRouteSupport extends HttpInstrumentation with WesCromwellRouteSupport {
         }
       )
     }
+
+  def toWesResponse(workflowId: WorkflowId, workflowState: WorkflowState):  WesRunStatus = {
+    WesRunStatus(workflowId.toString, WesState.fromCromwellStatus(workflowState))
+  }
+
+  def toWesResponseId(workflowId: WorkflowId): WesRunId ={
+    WesRunId(workflowId.toString)
+  }
+
+  def wesSubmitRequest(formData: Multipart.FormData, isSingleSubmission: Boolean): Route = {
+    def getWorkflowState(workflowOnHold: Boolean): WorkflowState = {
+      if (workflowOnHold)
+        WorkflowOnHold
+      else WorkflowSubmitted
+    }
+
+    def sendToWorkflowStore(command: WorkflowStoreActor.WorkflowStoreActorSubmitCommand, warnings: Seq[String], workflowState: WorkflowState): Route = {
+      // NOTE: Do not blindly copy the akka-http -to- ask-actor pattern below without knowing the pros and cons.
+      onComplete(workflowStoreActor.ask(command).mapTo[WorkflowStoreSubmitActor.WorkflowStoreSubmitActorResponse]) {
+        case Success(w) =>
+          w match {
+            case WorkflowStoreSubmitActor.WorkflowSubmittedToStore(workflowId, _) =>
+              completeResponse(StatusCodes.Created, toWesResponseId(workflowId), warnings)
+            case WorkflowStoreSubmitActor.WorkflowsBatchSubmittedToStore(workflowIds, _) =>
+              completeResponse(StatusCodes.Created, workflowIds.toList.map(toWesResponse(_, workflowState)), warnings)
+            case WorkflowStoreSubmitActor.WorkflowSubmitFailed(throwable) =>
+              throwable.failRequest(StatusCodes.BadRequest, warnings)
```

Comment:
Is this a valid WES 400 response?

I wouldn't necessarily hold up this PR for it, but be aware that the WES client might fail if pointed at an API that doesn't respond as it expects :)

Reply:
I had this same thought. I guess it's be best to change this to a Wes-like response to be on the safe side.